homeassistant/components/lupusec/binary_sensor.py

```diff
@@ -36,6 +42,20 @@ def setup_platform(
     add_entities(devices)
 
 
+async def async_setup_entry(hass, config_entry, async_add_devices):
+    """Set up a binary sensors for a Lupusec device."""
+
+    data = hass.data[LUPUSEC_DOMAIN]
```

Comment:
Can you reformat the import to just import DOMAIN instead of alias it?

Reply:
Done